docs/tools/locust/running-load-tests.md

```diff
@@ -3,15 +3,37 @@ sidebar_position: 2
 ---
 # Running Load Tests
 
-There are several ways you can run load tests. You can run `locust` locally or use the deployed version (on AWS). You 
+There are several ways you can run load tests. You can run `locust` locally or use the deployed version (on AWS). You
 can also use a local `mymove` server or the one deployed in the load test environment (on `loadtest.dp3.us`). One thing
-to note is that you can't point the deployed `locust` instance at a local `mymove` server. This means you have three 
-possible combinations:
-
-* Run `locust` locally against local `mymove` server (i.e. local -> local)
-* Run `locust` locally against load test env `mymove` server (i.e. local -> `loadtest.dp3.us`)
-* Run `locust` from AWS against the load test env `mymove` server (i.e. AWS -> `loadtest.dp3.us`)
-
+to note is that you can't point the deployed `locust` instance at a local `mymove` server.
+
+### Visualization of possible Locust combinations
+```mermaid
+flowchart RL
+    subgraph Environments
+        subgraph Local
+            direction TB
+            l_customer[(Customer App)]
+            l_office[(Office App)]
+            l_prime[(Prime API)]
+        end
+
+        subgraph AWS
```

Comment:
Do you think it'd be good to include the `loadtest.dp3.us` in here? To differentiate between this AWS and the AWS locust instance.

Reply:
I figured the differentiation would be that the AWS locust is outside of the "Environments > AWS". The reason I went with a short name here is because it's used in the diagram. Also, I didn't want to tie the name to an actual URL. It's possible our URL could change. And it's less likely that our IaaS would change. But it's not a terrible idea if you think it should read the URL. I'm okay with it either way.